docs/sql-data-sources-parquet.md

```diff
@@ -252,6 +252,42 @@ REFRESH TABLE my_table;
 
 </div>
 
+## Data Source Option
```

Comment:
Do you plan to list other datasource options here? If not, I would write it as another section like Rebasing Datetime.

Reply:
Yes, other options should be here.